### PR TITLE
Minor fix

### PR DESCRIPTION
**Pull request by akash2237778**

### CODE REVIEW

**Comment by lvarin** [comment]
Looks good to me. To be merged after https://github.com/elixir-cloud-aai/Pubgrade/pull/11 is merged

👍🏽

**Comment by akash2237778** [comment]
@lvarin I added a few more modifications. Can you please re-review this?

File: deployment/values.yaml [hunk 1]
```diff
@@ -14,16 +14,15 @@ volumes:
     # In case you are working with minikube or another single-worker solution
     # you can add a peristent volume from a local directory. For fully-distributed
     #clusters you should use a StorageClass already existing in your cluster, so set this to false.
-    deployLocalPv: false
-    pathToLocalDir: '/tmp/pubgrade-pv' # needs to be changed accordingly
+    deployLocalPv: true
+    pathToMountedDir: /pubgrade_temp_files
+    pathToLocalDir: /tmp/pubgrade-pv
```
Comment (lvarin) [inline deployment/values.yaml:19]: Why did you remove the comment?

File: deployment/values.yaml [hunk 2]
```diff
@@ -14,16 +14,15 @@ volumes:
...
     name: pubgrade-pvc
     storageClass: manual
     size: 2Gi
   mongodb:
-    deployLocalPv: false
-    pathToLocalDir: '/tmp/mongo-pv' # needs to be changed accordingly
-
+    deployLocalPv: true
+    pathToLocalDir: /tmp/mongo-pv
```
Comment (lvarin) [inline deployment/values.yaml:25]: Why did you remove the comment?

**Review by lvarin — COMMENTED** [review]
Looks good to me. The only comment I have is not critical. 

If you resolve the conflict that Github is complaining about, I will merge this PR

**Comment by uniqueg** [comment]
@akash2237778 Is this PR still active?

**Comment by akash2237778** [comment]
@lvarin Conflict resolved

**Review by lvarin — APPROVED** [review]
LGTM
